kong/plugins/opentelemetry/otlp.lua

```diff
@@ -72,9 +76,37 @@ local function transform_events(events)
   return pb_events
 end
 
+-- this function is to translate the span to otlp span, but of universal span format
+local function translate_span(span)
+  local trace_id = span.trace_id
+  local len = #trace_id
+  local new_id = trace_id
+
+  -- make sure the trace id is of 16 bytes
+  if len > 16 then
```

Comment:
We should not use the magic number `16`, what does it mean? 
Could we define a readable const for it?